returnn/datasets/basic.py

```diff
@@ -621,10 +621,12 @@ def _apply_partition_epoch(cls, seq_index, partition_epoch, epoch):
 
         return seq_index
 
-    def _get_random_seed_for_epoch(self, epoch, num_epochs_fixed=1):
+    def _get_random_seed_for_epoch(self, epoch, num_epochs_fixed=1, include_random_seed_offset=True):
```

Comment:
That's wrong. The `random_seed_offset` should always be used.

I think what you actually want to change is `_get_default_random_seed_offset`, not this function here.